src/theme/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from "./theme.directive";
-export * from "./theme.module";
```

Comment:
Could we still keep Theme module? Same as change as the other index files

Reply:
I actually had an issue with the following code since index file doesn't export Theme module:

```ts
import { ThemeModule } from 'carbon-components-angular';
```